2018-05-17T19:59:52.172+0000 I CONTROL  [initandlisten] MongoDB starting : pid=4752 port=27017 dbpath=/data/db 64-bit host=ip-172-31-38-83
2018-05-17T19:59:52.172+0000 I CONTROL  [initandlisten] db version v3.6.4
2018-05-17T19:59:52.172+0000 I CONTROL  [initandlisten] git version: d0181a711f7e7f39e60b5aeb1dc7097bf6ae5856
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.0-fips 29 Mar 2010
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten] modules: none
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten] build environment:
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten]     distmod: amazon
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten]     distarch: x86_64
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten] options: { config: "./mongodb/mongod.conf", net: { bindIp: "127.0.0.1", port: 27017 }, processManagement: { fork: true }, setParameter: { enableLocalhostAuthBypass: "false" }, storage: { journal: { enabled: true } }, systemLog: { destination: "file", logAppend: true, path: "./mongod.log" } }
2018-05-17T19:59:52.173+0000 I STORAGE  [initandlisten] exception in initAndListen: DBPathInUse: Unable to lock the lock file: /data/db/mongod.lock (Resource temporarily unavailable). Another mongod instance is already running on the /data/db directory, terminating
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten] now exiting
2018-05-17T19:59:52.173+0000 I CONTROL  [initandlisten] shutting down with code:100
